synapseclient/api/__init__.py

```diff
@@ -6,12 +6,50 @@
     post_entity_bundle2_create,
     put_entity_id_bundle2,
 )
-
+from .entity_services import (
```

Comment:
These API services are just splitting out functionality in the `client.py` around the various REST APIs. Since this is going to be replaced with the OpenAPI Generated client I am going to focus on integration testing higher level pieces than these individual level functions.